.github/workflows/test-templates.yaml

```diff
@@ -45,3 +45,25 @@ jobs:
         working-directory: charts/${{ matrix.manifest }}
         run: |
           helm template --dry-run --debug test . --set "global.skyfjell.validate.api=false" -f examples/values.test.yaml
+  test-libraries:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Install Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.8.1
+      - name: Prepare mock chart
+        working-directory: "charts"
```

Comment:
Do this in `$GITHUB_WORKSPACE/test` or `~/test`. Putting this in the `charts` path could result in the chart being released. Will result in like `file://../../charts/charts/common`